llvm/lib/Target/DirectX/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ add_llvm_target(DirectXCodeGen
   DXILResourceAccess.cpp
   DXILShaderFlags.cpp
   DXILTranslateMetadata.cpp
-
+  DXILRootSignature.cpp
   LINK_COMPONENTS
```

Comment:
Please keep the blank line between the sources and the LINK_COMPONENTS line - it makes this easier to read.